all data/src/battlepass_runner.py

```
@@ -0,0 +1,131 @@
+    common.click_matching("pictures/CustomAdded1080p/battlepass/exit.png")
+
+
+def main():
+    """Main function for battlepass runner"""
+    # Register signal handlers for clean exit if it is being run in the main thread
+    if threading.current_thread() is threading.main_thread():
+        signal.signal(signal.SIGTERM, signal_handler)
+        signal.signal(signal.SIGINT, signal_handler)
+
+    try:
+        logger.info(f"Starting battlepass rewards automation")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)